src/graph/transform/partition_hetero.cc

```diff
@@ -98,6 +98,32 @@ HaloHeteroSubgraph GetSubgraphWithHalo(std::shared_ptr<HeteroGraph> hg,
     }
   }
 
+  if (num_hops > 0) {
+    EdgeArray out_edges = hg->OutEdges(0, nodes);
```

Comment:
This is going to add halo nodes for out edges?

Reply:
yes